CHANGELOG.md

```diff
@@ -13,7 +13,8 @@
 - Added support for mitmproxy (via @etiennelb)
 - Added support for Hocus Focus (via @jrolfs)
 - Added support for TripMode (via @ttuygun)
-- Added support for Starship (via @callummr)
+- Added support for Starship (via @callumm)
```

Comment:
Why?

Reply:
Sorry, just a mistake.

Reply:
Can you fix it?

Reply:
I did

Reply:
No, the PR is still showing that you've changed `callummr` to `callumm`.